### DISCUSSION TITLE
Geomap: Speed

### DISCUSSION THREAD

**Discussion by UlrichThiess** [grafana/grafana#37377] [Dashboards, Panels & Visualizations]
Grafana v8.1.0-29738pre
on Windows Server 2019

So that TrackMap shows an exact route, 50,000 points are displayed per day. That's one point every 2 seconds.

This amount of points takes a long time at Geomap and is not usable. The query is fast but the display of the points is slow.

Please keep this in mind when you start implementing a path representation.

Thank you.

**Comment by nmarrs** [comment]
Hi @UlrichThiess, any updates on whether or not this issue is still occurring in Geomap in v9.2+? Thanks!

**Comment by leeoniya** [comment]
https://mourner.github.io/simplify-js/ can turn 73k points into a few hundred really quickly.

**Comment by UlrichThiess** [comment]
Now the performance quiet good. Thanks for this new panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
